projects/plugins/jetpack/_inc/client/traffic/related-posts.jsx

```diff
@@ -55,11 +58,166 @@ class RelatedPostsComponent extends React.Component {
 		this.updateOptions( 'show_thumbnails' );
 	};
 
-	trackConfigureClick = () => {
-		analytics.tracks.recordJetpackClick( 'configure-related-posts' );
+	handleAppendPostsToggleChange = () => {
+		this.updateOptions( 'append_to_posts' );
 	};
 
+	trackConfigureClick = type => {
+		analytics.tracks.recordJetpackClick( {
+			target: 'configure-related-posts',
+			type, // Classic for classic themes (customizer link), // blockTheme for block themes (site editor link).
+		} );
+	};
+
+	configureLink() {
+		const { siteAdminUrl, isBlockEnabledTheme, lastPostUrl } = this.props;
+
+		// For block themes, folks will want to add the Related Posts block directly in the site editor.
+		if ( isBlockEnabledTheme ) {
+			return (
+				<Card
+					compact
+					className="jp-settings-card__configure-link"
+					onClick={ this.trackConfigureClick( 'blockTheme' ) }
+					href={ `${ siteAdminUrl }site-editor.php` }
+				>
+					{ __( 'Add a Related Posts Block to your site.', 'jetpack' ) }
+				</Card>
+			);
+		}
+
+		const configureUrl =
+			siteAdminUrl +
+			'customize.php?autofocus[section]=jetpack_relatedposts' +
+			'&return=' +
+			encodeURIComponent( siteAdminUrl + 'admin.php?page=jetpack#/traffic' ) +
+			'&url=' +
+			encodeURIComponent( lastPostUrl );
+
+		return (
+			<Card
+				compact
+				className="jp-settings-card__configure-link"
+				onClick={ this.trackConfigureClick( 'classic' ) }
+				href={ configureUrl }
+			>
+				{ __( 'Configure related posts in the Customizer', 'jetpack' ) }
+			</Card>
+		);
+	}
+
+	classicRelatedPostsSettings() {
+		return (
+			<>
+				<CompactFormToggle
+					checked={ this.state.append_to_posts }
+					disabled={ this.props.isSavingAnyOption( [ 'related-posts', 'append_to_posts' ] ) }
+					onChange={ this.handleAppendPostsToggleChange }
+				>
+					<span className="jp-form-toggle-explanation">
+						{ __(
+							'Automatically display related posts to the bottom of each one of your posts.',
```

Comment:
Nit: This might be better phrased as "display related posts _at_ the bottom of each one of your posts.", but it may also be simpler to use fewer words by just saying "display related posts at the bottom of each post."
```suggestion
							'Automatically display related posts at the bottom of each one of your posts.',
```

Reply:
Done in 8c0ced8298baab03abc66b87f851b70f7a8a3abc